netebpfext/net_ebpf_ext.h

```diff
@@ -143,6 +143,22 @@ typedef struct _net_ebpf_extension_wfp_filter_context
     HANDLE wfp_engine_handle;  ///< WFP engine handle.
 } net_ebpf_extension_wfp_filter_context_t;
 
+#ifdef _DEBUG
```

Comment:
There is a lot of duplicate code here. Why?

Reply:
I refactored slightly - I don't know if there is a cleaner way to handle the #ifdef check inline in a more elegant way. Happy to try out more suggestions if you have any.